### ISSUE TITLE
Adding Peer Identity to labels

### ISSUE THREAD

**Issue by joseph-theiss** [tynany/frr_exporter#97]
Current:
```
frr_bgp_peer_state{afi="ipv4", ..., vrf="default"}
```

Suggested:
```
frr_bgp_peer_state{afi="ipv4", ..., vrf="default", identity="SomeCoolNewNode"}
```

Having and identity field will help identifying dynamic components in a DMVPN with hundreds of nodes on a grafana dashboard. 

And example frr command which links BGP IP, NMBA, Claimed NMBA, and Identity
`show ip nhrp cache`

**Comment by tynany** [comment]
Sorry for the delay in getting to this. Can you please provide a specific example of the feature you're requesting?

For example: "From the output of `show ip nhrp cache`, please use text $insertYourCriteria as the `identity` label."

**Comment by 4xoc** [comment]
FRR already gets fqdns through bgp capabilities (at least when the peer supports it too). This information is already easily available within vtysh. In my mind that would the the logical source for such label (maybe actually called `hostname` instead of `identity`). Happy to do this code change too.

**Comment by tynany** [comment]
This feature is now available in release [v1.2.0](https://github.com/tynany/frr_exporter/releases/tag/v1.2.0).
